orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/CloudwatchMetricsService.java

```diff
@@ -134,13 +136,26 @@ public void incrementAuthenticationSuccess(
         }
     }
 
-    public void incrementLogout(Optional<String> clientId) {
+    public void incrementLogout(
+            Optional<String> clientId,
+            Optional<AccountInterventionStatus> accountInterventionStatus) {
```

Comment:
Aesthetic thing (I personally really dislike using Optional as parameters, but that's a losing battle at this point):

Could we overload this, so we have incrementLogout(clientId) -> incrementLogout(clientId, empty)